aws_rules_s3_access_logs/aws_s3_access_error.yml

```diff
@@ -7,6 +7,7 @@ ResourceTypes:
   - AWS.S3ServerAccess
 Tags:
   - AWS
+  - Security Configuration
```

Comment:
How about Security Control? That might be a little more clear

Reply:
Easy enough.